beforeBuild.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const cp = require('child_process')
+
+// Rebuild native modules for ia32 and run webpack again for the ia32 part of windows packages
```

Comment:
Does this assume that the 64-bit build always runs before the 32-bit build? (and if so, is that guaranteed to happen?)

Reply:
Yes and yes unless you specify otherwise.
This was to save some build time on the ci.
I'll change it it rebuild in all cases.